src/pages/module-deactivation/index.js

```diff
@@ -128,7 +128,7 @@ const ModuleDeactivation = () => {
       record: JSON.stringify(resultObject)
     })
       .then(res => {
-        toast.success('Record Generated Successfully')
+        toast.success('Record Updated Successfully')
       })
       .catch(error => {})
```

Comment:
we should follow error message decision when reached

Reply:
i can't put comment above so i will put all here below:
 use form instead of use formik